### PR TITLE
[Tests-only] Enable tests for single file share

### PR DESCRIPTION
**Pull request by PVince81**
Prove that https://github.com/owncloud/core/pull/37610 works

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -1,7 +1,7 @@
 def main(ctx):
   before = [
     testing(ctx),
-    apiTests(ctx, 'master', 'ad274cb7f2a20ffba7fb9c65726ae9ef9270e4c0'),
+    apiTests(ctx, 'master', '24406db4f742b174fcd40740e6938a463eb5ebd2'),
```
Comment (individual-it) [inline .drone.star:4]:
```suggestion
    apiTests(ctx, 'enable-tests-ocis-reva-278', ''),
```

**Comment by PVince81** [comment]
rebased and adjusted

**Comment by PVince81** [comment]
hmmm... strange
```

/srv/app/testrunner/tests/acceptance/features/apiSharePublicLink1/accessToPublicLinkShare.feature:10
/srv/app/testrunner/tests/acceptance/features/apiSharePublicLink1/accessToPublicLinkShare.feature:20
/srv/app/testrunner/tests/acceptance/features/apiSharePublicLink1/accessToPublicLinkShare.feature:30
/srv/app/testrunner/tests/acceptance/features/apiSharePublicLink1/accessToPublicLinkShare.feature:43


```

**Comment by PVince81** [comment]
test commit to be updated again in a separate PR
won't do it right away as I intend to raise more test-enabling PRs and doing it for each and every one is boring

**Pull request closed without merging**